examples/basic_voice.py

```diff
@@ -112,9 +115,6 @@ async def stop(self, ctx):
 
         await ctx.voice_client.disconnect()
 
-    @play.before_invoke
-    @yt.before_invoke
-    @stream.before_invoke
```

Comment:
This example was about pre-invoke hooks, now it's about something else

Reply:
I have no idea what the fuck is going on in the hook decorators but it was complaining so I just yeeted it.